src/instructlab/cli/process/attach.py

```diff
@@ -0,0 +1,24 @@
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+
+
+@click.command(name="attach")
+@clickext.display_params
+@click.option("--uuid", type=click.STRING, help="PID of the process to attach to")
```

Comment:
Is this a true Linux PID or an InstructLab-generated UUID?

Reply:
instructlab generated UUID. using the pid as a tracker doesn't make much sense given that most of these processes spawn children and those ones are what we are really interested in

Reply:
So the help text is wrong then?